packages/@okta/i18n/src/properties/login.properties

```diff
@@ -483,6 +483,7 @@ oktaverify.send = Send Push
 oktaverify.resend = Re-send Push
 oktaverify.sent = Push sent!
 oktaverify.rejected = You have chosen to reject this login.
+oktaverify.rejected.upgradeRequired = Verification failed because your Okta Verify version is no longer supported. To sign in, please update Okta Verify, then try again.
```

Comment:
would it be helpful to show app versions to user ?

Reply:
@stevennguyen-okta @ManuMalhotra-okta  thoughts ?

Reply:
Interesting idea, I discussed this with Aakash and we think that it wouldn't add much because the user would just have to go update anyway, and knowing what version number wouldn't change anything since presumably the update would fix whatever the reason is for updating.

Reply:
nit: @stevennguyen-okta should we explicitly mention they need to update the app? Not sure if users might think of something like "admin needs to update my Okta Verify version" as opposed to "I need to go to the app store and update my app". e.g.
```
Your Okta Verify app is no longer supported. To sign in, please update Okta Verify, then try again.
```

Reply:
Good idea, we originally had a title on the infobox that kinda helped but decided not to hack the infobox so that message was lost. I've updated the mocks and the message should now mention "please update Okta Verify on {appStoreName}". So if you're on iOS it'd be "on the App Store" and Android would be "on Google Play"